azavea/pfb-network-connectivity
# Fix build failures from Jessie sunset

## Overview

Updates base images to ones based on Debian Stretch, since the support window for Jessie has ended and the apt-get updates started failing.

This caused some drift in the analysis container due to newer versions of a couple things behaving differently than the versions we had built the scripts around.  I didn't detect any regressions in the AngularJS or Tilegarden containers.

### Demo

![image](https://user-images.githubusercontent.com/6598836/55807676-9611be80-5ab0-11e9-9fea-8261db0b7d9b.png)
(no way to actually tell, but that's a job run locally with the updated analysis container and being displayed via the new AngularJS and Tilegarden containers)

### Notes

- I tried to upgrade the Tilegarden container to Stretch without upgrading the Node version, since the Lambda runtime is 8.10, so running 8.15 locally seems not as good.  But there's no such image.  Also, the node environment in the official Node project images was never identical to the Lambda environment anyway, so I don't think being a couple minor versions ahead is likely to be noticeable.

## Testing Instructions

- Run `./scripts/update` inside the VM.  It now includes `--pull` in the `docker-compose build` commands, so it will pull new base images.  This takes a while.
- Run an analysis.  It should finish successfully and the results should show up in the front-end as usual.

## Checklist


Resolves #738.
